db/primary_migrate/20230809194211_add_reminder_sent_at_to_usps_confirmation_codes.rb

```diff
@@ -0,0 +1,8 @@
+class AddReminderSentAtToUspsConfirmationCodes < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
+
+  def change
+    add_column :usps_confirmation_codes, :reminder_sent_at, :datetime, precision: nil
+    add_index :usps_confirmation_codes, :reminder_sent_at, algorithm: :concurrently
```

Comment:
Wondering if we need an index here. Are we querying the usps_confirmation_codes table by when reminders were sent?